packages/falcon-sensor/overlay/questions.yaml

```diff
@@ -0,0 +1,98 @@
+questions:
+  - variable: node.image.repository
+    description: "URL of container image repository holding containerized Falcon sensor. Defaults to 'falcon-node-sensor'."
+    required: true
+    type: string
+    default: falcon-node-sensor
+    label: Container Image Repository
+    group: "Node Container Images"
+
+  - variable: node.image.tag
+    description: "Container registry image tag. Defaults to 'latest'."
+    required: true
+    type: string
+    default: "latest"
+    label: Container Image Tag
+    group: "Node Container Images"
+      
+  - variable: falcon.cid
+    description: "CrowdStrike Customer ID (CID). Passed to falconctl as \"--cid=\"{uuid string}\"\""
+    required: true
+    type: string
+    label: --cid
+    group: "Falcon Sensor Node Settings"
+
+  - variable: falcon.apd
+    description: "Description goes here. Passed to falconctl as \"--apt=true\" or \"--apt=false\"."
+    required: false
+    type: boolean
+    default: false
+    label: --apt
+    group: "Falcon Sensor Node Settings"
+
+  - variable: falcon.aph
+    description: "App Proxy Hostname (APH). Uncommon in container-based deployments. Passed to falconctl as \"--aph <app proxy host name>\""
+    required: false
+    type: string
+    label: --aph
+    group: "Falcon Sensor Node Settings"
+
+  - variable: falcon.app
+    description: "App Proxy Port (APP). Uncommon in container-based deployments. Passed to falconctl as \"--app=<app proxy port>\""
+    required: false
+    type: string
+    label: --app
+    group: "Falcon Sensor Node Settings"
+
+  - variable: falcon.trace
+    description: "Set trace level. Options are [none|err|warn|info|debug]. Passed to falconctl as \"--trace=[none|err|warn|info|debug]\""
+    required: false
+    type: string
+    label: --trace
+    default: "none"
+    group: "Falcon Sensor Node Settings"
+
```

Comment:
This one could be something like this:
```
  - variable: falcon.trace
    description: "Logging levels are none, err, warn, info, or debug (Same as falconctl --trace=[level])"
    required: false
    type: string
    label: Set logging trace level
    default: "none"
    group: "Falcon Sensor Node Settings"
```

Reply:
Thanks @PennyScissors! This should be fixed.